gwardia/templates/gwardia/index.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'core/base.html' %}
```

Comment:
`core/base.html` is just black background, so `gwardia/index.html` is just black background with text in left upper corner, did you want this to look like this??

![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/052c797b-1178-471d-bf8d-0bca337803a6)